velox/core/PlanNode.h

```diff
@@ -532,6 +532,27 @@ class AggregationNode : public PlanNode {
       bool ignoreNullKeys,
       PlanNodePtr source);
 
+  /// @param globalGroupingSets If the AggregationNode is over a GroupId node
+  /// then its possible that the grouping sets include global grouping sets
+  /// which have all null keys except for the groupId. These are for the global
+  /// aggregate values.
+  /// @param groupId If the AggregationNode is over a GroupId node,
```

Comment:
nit: Group ID key produced by the preceding GroupId node. Must be set if globalGroupingSets is not empty. Must not be set otherwise. Must be one of the groupingKeys.